gsa/src/web/entities/withEntitiesContainer.js

```diff
@@ -107,10 +107,13 @@ const withEntitiesContainer = (
       loadedFilter = filter;
     }
 
+    const reloadFunc = isDefined(loadEntities)
+      ? (newFilter = filter) => loadEntities(newFilter)
+      : undefined;
     return (
       <Reload
         reloadInterval={() => reloadInterval(props)}
-        reload={(newFilter = filter) => loadEntities(newFilter)}
+        reload={reloadFunc}
```

Comment:
If reloadFunc is undefined, would that cause a problem with the Reload component?

Reply:
Nope. See https://github.com/greenbone/gsa/blob/hyperion/gsa/src/web/components/loading/reload.js#L223 no reloading is done if the function is undefined.